content/content-tool/tool/src/webapp/vm/resources/sakai_dropbox_multiple_folders_upload.vm

```diff
@@ -186,6 +189,21 @@
 	{
 		const fileField = document.getElementById('MultipleFolderContent');
 		const nameField = document.getElementById('MultipleFolderDisplayName');
+                const feedback = document.getElementById('fileUploadFeedback');
+
+                if (fileField.files.length > 0)
+                {
+                        const fileName = fileField.files[0].name;
+                        const successTemplate = `$tlang.getString('label.success')`.replace("{0}", fileName);
+                        feedback.textContent = successTemplate;
+                        feedback.classList.remove('d-none');
+                }
+                else
+                {
+                        feedback.classList.add('d-none');
+			nameField.value = '';
+                }
```

Comment:
Do we need  `else` block ? when there's no file selected nothing changes and `d-none` is already present.

Reply:
I checked that when I deselect a file, the message and display name do not disappear. That's why I added the else block and it works as intended.

Reply:
@kunaljaykam can you please review this and let me know if any changes are required?

Reply:
You are correct. I tested your pr. The `else` block is necessary to clear outdated file info when the file picker is opened but no new file is selected. Without it, the old file name and feedback message remain and causes  console error.

I think we should re-write the whole `handleFileChange` function it looks messy, and include the display name field inside; fileField.files.length > 0)  check as well so display name gets updates as well.